app/src/main/java/com/coursein65apps/mynotfirstappkotlin/ContactDetailsFragment.kt

```diff
@@ -24,4 +34,24 @@ class ContactDetailsFragment : Fragment(R.layout.item_fragment_details) {
         }
     }
 
+    private fun getPersonContact(id: Int) {
+        CoroutineScope(Dispatchers.IO).launch {
+            val contact = (activity as MainActivity).getContactService().getContact(id)
+            withContext(Dispatchers.Main) {
+                setPersonData(contact)
+            }
+        }
+    }
+
+
+    private fun setPersonData(contact: Contact) {
+        viewBinding.fullName.text = contact.name
```

Comment:
nit: 
        with (viewBinding) {
        fullName.text = contact.name
        description.text = contact.description
        ...
        contact.photoContact.let { photoContact.setImageResource(it) }
        }